charts/datadog/values.yaml

```diff
@@ -1135,9 +1135,13 @@ agents:
     rollingUpdate:
       maxUnavailable: "10%"
 
-  # agents.priorityClassName -- Sets PriorityClassName if defineds
+  # agents.priorityClassName -- Sets PriorityClassName if defined
   priorityClassName:
 
+  # agents.priorityClassValue -- Value used to specify the priority of the scheduling of Datadog Agent's Daemonset pods.
+  ## The PriorityClass uses PreemptLowerPriority.
+  priorityClassValue: 1000000000
```

Comment:
the value seems huge, does the kubernetes documentation have some advice on the possible value? Like from 200000000 to 100000000 for CNI/system pods

Reply:
I did not find doc, but in Autopilot specifically the value is lower than the value in the priority classes for system pods (critical-cluster and critical-node), which is 2000000000. As the priorityClass has the config to preempt the lower priority it means that we will be scheduled after the kube-system apps but before the applications (or anything < 1000000000).